keyboards/melgeek/z70ultra/z70ultra.h

```diff
@@ -19,19 +19,19 @@
 #include "quantum.h"
 
 #define XXX KC_NO
-
+#ifdef REV1
```

Comment:
these should go into the revision folders, not here.

Reply:
I also tried some methods, but they failed to compile, can you give me some guidance?

Reply:
`keebio/iris` should be a good example.